js/src/tooltip.js

```diff
@@ -734,10 +734,6 @@ class Tooltip {
     this.config.animation = initConfigAnimation
   }
 
-  _getCustomClass() {
-    return this.element.getAttribute('data-custom-class') || this.config.customClass
```

Comment:
How is this not needed? 😵

Reply:
Because the `customClass` will be in the configuration already, even if it is provided in the data attributes 🤔 
https://github.com/twbs/bootstrap/blob/2d8c02eb1024ef2ce8877d6484a9ea8a4d9cd67f/js/src/tooltip.js#L660-L664

Reply:
And jQuery's [`addClass()`](https://api.jquery.com/addclass/) method will handle it, either `this.config.customClass` is a function or a class name (Even it is a blank string)

Reply:
yep good catch 👌